tests/v3_api/test_upgrade.py

```diff
@@ -16,6 +16,7 @@
 namespace = {"p_client": None, "ns": None, "cluster": None, "project": None,
              "testclient_pods": []}
 upgrade_check_stage = os.environ.get('RANCHER_UPGRADE_CHECK', "preupgrade")
+validate_ingress = os.environ.get('RANCHER_INGRESS_CHECK', "True")
```

Comment:
I think we only need to do the `ast.literal_eval()` once, after we get the environment variable, since it will not be changed after execution starts.

```
validate_ingress = ast.literal_eval(os.environ.get('RANCHER_INGRESS_CHECK', "True"))
```
or
```
validate_ingress = os.environ.get('RANCHER_INGRESS_CHECK', "True")
validate_ingress = ast.literal_eval(validate_ingress)
```

Then we can do a regular check -- `if validate_ingress:`

Reply:
@bmdepesa Yes. I will do this change.